tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,15 @@
+Exit Code: 1
+Standard output:
+lib/webpack.js(64,9): error TS2339: Property 'version' does not exist on type 'typeof import("/webpack/webpack/lib/webpack")'.
```

Comment:
@sandersn are you aware of these issues?

Reply:
Yes, fixed in #24871